Umbraco forms validation update (#2501)
umbraco/UmbracoDocs: Add-ons/UmbracoForms/Developer/Prepping-Frontend/index.md

# Preparing your frontend

> In order for Umbraco Forms to work correctly, Umbraco Forms need you to include some client dependencies.

Reviewer: ```suggestion
In order for Umbraco Forms to work correctly, you need to include some client dependencies.
```

> With latest version of Umbraco Forms, we provide a method that will output the script tags containing the dependencies. Dne with this razor method:

Reviewer: ```suggestion
Use the following Razor method to output script tags containing the dependencies:
```

```html
<head>
    @Html.RenderUmbracoFormDependencies()
</head>
```

> All dependencies originates from your UmbracoForms instalation, no external references.

Reviewer: ```suggestion
All dependencies originates from your Umbraco Forms installation, which means that no external references are needed.
```

> If you like to use jQuery instead, you will instead have to manually add those. Without using the above. Jump the the section 'Use jQuery' to learn more.

Reviewer: ```suggestion
If you like to use jQuery instead, you can manually add those. Without using the above, jump the 'Use jQuery' section below to learn more.
```
